sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationOperation.cs

```diff
@@ -185,10 +187,11 @@ internal DocumentTranslationOperation(DocumentTranslationRestClient serviceClien
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> used for the periodical service calls.</param>
         /// <returns>The last HTTP response received from the server.</returns>
         /// <remarks>
-        /// This method will periodically call UpdateStatusAsync till HasCompleted is true, then return the final result of the operation.
+        /// This method will periodically call UpdateStatusAsync till HasCompleted is true.
+        /// An API call is then made to retrieve the status of the documents.
         /// </remarks>
         public override ValueTask<Response<AsyncPageable<DocumentStatusDetail>>> WaitForCompletionAsync(CancellationToken cancellationToken = default) =>
-            this.DefaultWaitForCompletionAsync(cancellationToken);
+            WaitForCompletionAsync(OperationHelpers.DefaultPollingInterval, cancellationToken);
```

Comment:
Consider setting the polling interval to  `30s`  until we address https://github.com/Azure/azure-sdk-for-net/issues/19442